src/main/java/Duke.java

```diff
@@ -1,10 +1,87 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    //TODO make methods for the different operations
+    protected static Task[] list = new Task[100];
+    public static final String OPENING_MSG = "Hello! I'm Dukebot\n\tWhat can I do for you?";
+    public static final String CLOSING_MSG = "Goodbye! Hope to see you again soon ^^!";
+    public static final String HORIZONTAL = "---------------------------------";
+    public static final String MARK_MSG = "Nice! I've marked this task as done:";
+    public static final String UNMARK_MSG = "Oki! I've marked this task as not done yet:";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println(HORIZONTAL + "\n\t" + OPENING_MSG + "\n" + HORIZONTAL);
+
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine().toLowerCase();
+
+        int listNum;
+
+        int item = 0;
+
+        while (!line.equals("bye")) {
+            if (!(line.equals("list") || line.startsWith("mark") || line.startsWith("unmark"))) {
+                list[item] = new Task(line);
+                System.out.println(HORIZONTAL + "\n\tadded: " + list[item].description + "\n" + HORIZONTAL);
+                item++;
+            }
+
+            line = in.nextLine().toLowerCase();
+
+            // display list
+            if (line.equals("list")) {
+                System.out.println(HORIZONTAL + "\n\tHere are the tasks in your list:");
+                for (int i = 0; i < item; i++) {
+                    System.out.println("\n\t" + (i + 1) + ". [" + list[i].getStatusIcon() + "] " + list[i].description + "\n");
+                }
+                System.out.println(HORIZONTAL);
+            }
+
+            // Mark item as done: constraint - user input begins with "mark"
+            if (line.startsWith("mark")) {
+
+                StatusToggle(line, item, MARK_MSG, true);
+
+            }
+
+            // Unmark item: constraint - user input begins with "mark"
+            if (line.startsWith("unmark")) {
+
+                StatusToggle(line, item, UNMARK_MSG, false);
```

Comment:
Please rename it to `toggleStatus()`. Method should look like a verb and start with lowercase